doc/source/myrocks/limitations.rst

```diff
@@ -90,3 +90,11 @@ You should also consider the following:
      |MySQL| Documentation: Preparing Your Installation for Upgrade
         https://dev.mysql.com/doc/refman/8.0/en/upgrade-prerequisites.html
 
+* |Percona Server| 8.0 and Unicode 9.0.0 standards have defined a change in the
+  handling of binary collations. These collations are handled as NO PAD,
+  trailing spaces are included in comparisons. A binary collation comparison
+  may result in two unique rows inserted and does not generate a`DUP_ENTRY`
+  error. MyRocks key encoding and comparison does not account for this
+  character set attribute. Older indexes, created under Version 5.7 and
+  early versions of 8.0 prior to 8.0.13-3 must be rebuilt to return
```

Comment:
Drop the "Older versions ..." as it will only apply once we actually fix this issue, which we haven't yet.

Reply:
Updated based on comments.